src/content/guides/production.md

```diff
@@ -169,6 +169,7 @@ __webpack.prod.js__
   const common = require('./webpack.common.js');
 
   module.exports = merge(common, {
+    mode: 'production',
 +   devtool: 'source-map',
     plugins: [
 -     new UglifyJSPlugin()
```

Comment:
You have to; `-require(UglifyJSPlugin)` too.

Reply:
It's not related to your PR, but lines 204-205 are strange...

Reply:
That's how the docs are presenting code changes from the previous step, it just looks weird on GitHub. 

![captura de pantalla 2018-06-20 a las 15 05 59](https://user-images.githubusercontent.com/3504472/41684715-8e12d47c-749b-11e8-90de-59bce0bef61c.png)

Reply:
Also `UglifyJSPlugin` is already required on lines 168 and 195.

Reply:
Technically it's not removing brackets, it's just adding a comma. That's how git sees it. If you add a comma git takes the whole line as modified.

Reply:
Ah ok, it means, replace this with that.  You are right. I think it's enough for today ;-)